test/examples/expressions_explorer/index.ts

```diff
@@ -18,6 +18,7 @@ export default function ({
   const PageObjects = getPageObjects(['common', 'header']);
 
   describe('expressions explorer', function () {
+    this.tags('ciGroup1');
```

Comment:
Can we set these to ciGroup2 or ciGroup3? ciGroup1 is already one of the longest-running.

Reply:
improved 👍 
it just was before part of `ciGroup1`